hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestUtils.java

```diff
@@ -107,6 +111,28 @@ public static HoodieTableMetaClient init(Configuration hadoopConf, String basePa
     return HoodieTableMetaClient.initTableAndGetMetaClient(hadoopConf, basePath, properties);
   }
 
+  public static final void createMetadataFolder(String basePath) {
```

Comment:
Please check out `org.apache.hudi.common.testutils.FileCreateUtils`; we have been trying to move methods out from this class as it was crammed with all sorts of helpers.

Reply:
Thanks for you review,  since it's no need in this PR, I will remove it.